### PR TITLE
Add Syntax Highlighting to Interactive Prompt

### LINKED ISSUE #87: Syntax highlighting for Interactive Prompt

### PR DESCRIPTION
**Pull request by ZhiHanL**
Also make application case insensitive
closes #87

### CODE REVIEW

File: src/lib/tokenizer.ts [hunk 1]
```diff
@@ -19,6 +19,7 @@ class Tokenizer {
   tokenize(): void {
     let tokenizedProgram = Tokenizer.program;
     tokenizedProgram = tokenizedProgram.replace(/\n/g, "_");
+    tokenizedProgram = tokenizedProgram.toLowerCase();
```
Comment (john-zou) [inline src/lib/tokenizer.ts:22]: I think a tricky issue with this that the user would lose their capital letters in their custom strings. How should we deal with this? One way would be to change the title to "goi - the all lowercase flash cards app" (kidding)

Comment (ZhiHanL) [reply src/lib/tokenizer.ts:22]: Hmm right, let me figure something out

**Comment by ZhiHanL** [comment]
Modified approach only converts literals to lowercase :smile: @john-zou

**Comment by john-zou** [comment]
Wow it's that easy huh!! Nice!

**Comment by john-zou** [comment]
Wait, is highlight in master though?

**Pull request merged**